package.json

```diff
@@ -11,21 +11,24 @@
     "@material-ui/icons": "4.9.1",
     "@material-ui/lab": "4.0.0-alpha.56",
     "@material-ui/pickers": "3.2.10",
-    "material-ui-phone-number": "2.2.6",
     "bokehjs": "0.12.9",
     "check-dependencies": "1.1.0",
     "clsx": "1.1.1",
+    "core-js": "3.6.5",
     "css-loader": "4.2.0",
     "dayjs": "1.8.31",
     "exports-loader": "0.7.0",
     "imports-loader": "0.8.0",
+    "material-ui-phone-number": "2.2.6",
```

Comment:
Should these changes be in here?

Reply:
Yes, without these extra side dependencies, npm installs fail.